src/test/java/org/broadinstitute/hellbender/tools/sv/cluster/SVCollapserTest.java

```diff
@@ -0,0 +1,783 @@
+        Assert.assertEquals(collapser.collapseGenotypeAttributes(genotypes), expectedAttributes);
+
+        // Test as variant attributes
+        final List<SVCallRecord> variants = IntStream.range(0, inputAttributesList.size())
+                .mapToObj(i -> SVTestUtils.newNamedDeletionRecordWithAttributes(variantIds.get(i), inputAttributesList.get(i)))
+                .collect(Collectors.toList());
+        final Map<String, Object> expectedAttributesWithMembers = new HashMap<>(expectedAttributes);
+        expectedAttributesWithMembers.put(GATKSVVCFConstants.CLUSTER_MEMBER_IDS_KEY, variantIds);
+        Assert.assertEquals(collapser.collapseVariantAttributes(variants), expectedAttributesWithMembers);
+    }
+
+    @DataProvider(name = "collapseLengthTestData")
+    public Object[][] collapseLengthTestData() {
+        return new Object[][]{
+                {
+                        new int[]{1000},
+                        new String[]{"chr1"},
+                        new StructuralVariantType[]{StructuralVariantType.DEL},
+                        1,
+                        1000,
+                        StructuralVariantType.DEL,
+                        1000
+                },
+                {
+                        new int[]{1000, 1000},
+                        new String[]{"chr1", "chr1"},
+                        new StructuralVariantType[]{StructuralVariantType.DUP, StructuralVariantType.DUP},
+                        1,
+                        1000,
+                        StructuralVariantType.DUP,
+                        1000
+                },
+                {
+                        new int[]{300, 400},
+                        new String[]{"chr1", "chr1"},
+                        new StructuralVariantType[]{StructuralVariantType.DEL, StructuralVariantType.DUP},
+                        1001,
+                        1350,
+                        StructuralVariantType.CNV,
+                        350
+                },
+                {
+                        new int[]{300, 400},
+                        new String[]{"chr1", "chr1"},
+                        new StructuralVariantType[]{StructuralVariantType.INS, StructuralVariantType.INS},
+                        1,
+                        1,
+                        StructuralVariantType.INS,
+                        300
+                },
+                {
+                        new int[]{300, 400, 500},
+                        new String[]{"chr1", "chr1", "chr1"},
+                        new StructuralVariantType[]{StructuralVariantType.INS, StructuralVariantType.INS, StructuralVariantType.INS},
+                        1,
+                        1,
+                        StructuralVariantType.INS,
+                        400
+                },
+                {
+                        new int[]{-1},
+                        new String[]{"chr2"},
+                        new StructuralVariantType[]{StructuralVariantType.BND},
```

Comment:
Can you add another BND here or another case with multiple?

Reply:
Done